docs/source/WritingTests.rst

```diff
@@ -992,6 +992,82 @@ Now, trying to list the tests on the ``mytest.py`` file again::
 You can also use the ``:avocado: disable`` tag, that works the opposite way:
 Something looks like an Avocado test, but we force it to not be listed as one.
 
+Unittest Compatibility Limitations And Caveats
+==============================================
+
+The Avocado test runner uses a different approach to running tests
+than most other Python unittest test runners.  This brings some
```

Comment:
As discussed before, this is a style that I acquired earlier.  A good source of explanation on the reasons for using double-space is here:

http://creativepro.com/double-space-or-not-double-space/

IMHO, it helps when reading the sources, and it doesn't change anything when using the formatted version.  I'm not sure we should force a style for something that small at this point...